src/Initializer/MemoryManager.cpp

```diff
@@ -95,7 +95,19 @@ void seissol::initializers::MemoryManager::initialize()
   // initialize global matrices
   GlobalDataInitializerOnHost::init(m_globalDataOnHost, m_memoryAllocator, MEMKIND_GLOBAL);
   if constexpr (seissol::isDeviceOn()) {
-    GlobalDataInitializerOnDevice::init(m_globalDataOnDevice, m_memoryAllocator, memory::DeviceGlobalMemory);
+    bool serialize = false;
+    if (const char* value = std::getenv("SEISSOL_SERIALIZE_NODE_DEVICE_INIT")) {
+      serialize = strcmp(value, "1") == 0;
+    }
+    if (serialize) {
+      logInfo(MPI::mpi.rank()) << "Serializing device global data initialization.";
```

Comment:
I would use a different term as serializing for me rather means "serializing the data". Something like "Using serial mode for device global data initialization."?